src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1058,6 +1072,29 @@ static bool isOrContainsErrorType(TypeSymbol typeSymbol)
             }
         }
 
+        /// <summary>
+        /// Return true if <paramref name="overridingReturnType"/> is valid for the return type of an override method when the overridden method's return type is <paramref name="overriddenReturnType"/>.
+        /// </summary>
+        private bool IsValidOverrideReturnType(Symbol overridingSymbol, TypeWithAnnotations overridingReturnType, TypeWithAnnotations overriddenReturnType, DiagnosticBag diagnostics)
```

Comment:
nit: could be a local function

Reply:
I expect this will be useful elsewhere later.

---
In reply to: [413985145](https://github.com/dotnet/roslyn/pull/43576#discussion_r413985145) [](ancestors = 413985145)